core/app/c/[communitySlug]/settings/tokens/PermissionField.tsx

```diff
@@ -328,7 +414,7 @@ export const ConstraintFormFieldRender = ({
 	return (
 		<FormItemWrapper
 			dataTestId={`${scope}-${type}-checkbox`}
-			checked={Boolean(field.value)}
+			checked={typeof field.value === "object" ? "indeterminate" : Boolean(field.value)}
```

Comment:
this makes the checkbox have a `-` in it instead of also looking like a checkmark when a restriction is selected

![image](https://github.com/user-attachments/assets/dcb1b8b8-e29b-444a-8a95-d7a4b1087300)